src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -15,6 +15,10 @@
             <GroupBox x:Uid="AnalysisGroupBox"
                       Header="{x:Static local:AdvancedOptionPageStrings.Option_Analysis}">
                 <StackPanel>
+                    <CheckBox x:Name="Enable_power_save_mode"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Enable_power_save_mode}" />
```

Comment:
how would a user figure out what this means?  My view on power saving is not "we are doing less".  Rather, it's "we do less work when the machine doesn't have a lot of CPU avaliable".  i.e. it's "battery saving mode".  This is how phones work for example.  When you go into battery saving mode, htey do less things automatically, but you can still do anything you want explicitly.  I think we should model after that.